components/prism-wgsl.js

```diff
@@ -0,0 +1,94 @@
+Prism.languages.wgsl = {
+    'comment': {
+        pattern: /\/\/.*|\/\*[\s\S]*?(?:\*\/|$)/,
+        greedy: true,
+    },
+    'attributes': {
+        pattern: /(@)(?:(?:[_\p{XID_Start}][\p{XID_Continue}]+)|(?:[\p{XID_Start}]))/u,
```

Comment:
Prism still supports browsers that don't support the u flag, so please don't use it.

While a lot more simplistic, if we only focus on ASCII text, the equivalent non-Unicode pattern would be:
```suggestion
        pattern: /(@)[_a-z]\w*/i,
```